php/class-post-type.php

```diff
@@ -424,8 +440,12 @@ public function render_data_metabox( $post ) {
 				esc_url( $frontend_view_url ),
 				esc_html__( 'Preview Changeset', 'customize-snapshots' )
 			);
-
 			echo $fork_markup; // WPCS: XSS ok.
+			if ( $should_allow_split ) {
+				$split_text = __( 'Split', 'customize-snapshots' );
+				echo sprintf( '<button id="split-activate" class="button button-secondary" data-cancel-text="%s" data-original-text="%s">%s</button>',
+				esc_html__( 'Cancel Split', 'customize-snapshots' ), esc_attr( $split_text ), esc_html( $split_text ) );
```

Comment:
Let's reformat the `sprintf()` here so that each of its args are on a separate line.